PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/Activator.java

```diff
@@ -153,6 +155,8 @@ private void initAzureToolsCoreLibsSettings() {
             final String baseFolder = FileUtil.getDirectoryWithinUserHome(AZURE_TOOLS_FOLDER).toString();
             final String deprecatedFolder = FileUtil.getDirectoryWithinUserHome(AZURE_TOOLS_FOLDER_DEPRECATED).toString();
             CommonSettings.setUpEnvironment(baseFolder, deprecatedFolder);
+            Azure.az().config().setCloud(
```

Comment:
why convert string to azure environment and convert it back here?

Reply:
because the name of legacy code is not standard form, eg: GLOBAL